README.md

```diff
@@ -3,12 +3,7 @@
 # OpenHTF
 The open-source hardware testing framework.
 
-[![Build Status](https://travis-ci.org/google/openhtf.svg?branch=master)](https://travis-ci.org/google/openhtf)
```

Comment:
Same for me. Only the issuestats ones are broken. I'm in favor of leaving the Travis and Coveralls ones in, and only removing the broken ones.